### PR TITLE
{data,math}[intel/2021b] mkl-service v2.3.0, PyMC3 v3.11.1, Theano v1.1.2 w/ Python 3.9.6

### PR DESCRIPTION
**Pull request by ItIsI-Orient**
(created using `eb --new-pr`)
* https://github.com/easybuilders/easybuild-easyconfigs/pull/14809
* https://github.com/easybuilders/easybuild-easyconfigs/pull/14810
* https://github.com/easybuilders/easybuild-easyconfigs/pull/14890

### CODE REVIEW

**Comment by ItIsI-Orient** [comment]
Test report by @ItIsI-Orient
**FAILED**
Build succeeded for 3 out of 4 (4 easyconfigs in total)
node3202.victini.os - Linux UNKNOWN UNKNOWN, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/4638ce0737fa9a250c6967ca36eb2d20 for a full test report.

**Comment by ItIsI-Orient** [comment]
Test report by @ItIsI-Orient
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
node3202.victini.os - Linux UNKNOWN UNKNOWN, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/56a26bf43a95b79558e33a755fee7c6f for a full test report.

File: easybuild/easyconfigs/p/PyMC3/PyMC3-3.11.1-intel-2021b.eb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    ('tqdm', '4.62.3'),
+    ('netcdf4-python', '1.5.7'),
+    ('xarray', '0.20.1'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('typing_extensions', '3.10.0.2', {
```
Comment (boegel) [inline easybuild/easyconfigs/p/PyMC3/PyMC3-3.11.1-intel-2021b.eb:30]: Use a proper dependency for this instead? See #14809

File: easybuild/easyconfigs/p/PyMC3/PyMC3-3.11.1-intel-2021b.eb [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        'checksums': ['49f75d16ff11f1cd258e1b988ccff82a3ca5570217d7ad8c5f48205dd99a677e'],
+    }),
+    ('arviz', '0.11.4', {
```
Comment (boegel) [inline easybuild/easyconfigs/p/PyMC3/PyMC3-3.11.1-intel-2021b.eb:33]: Also here, use a proper dependency, see #14810

File: easybuild/easyconfigs/p/PyMC3/PyMC3-3.11.1-intel-2021b.eb [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        'checksums': ['01872758eeabb9941479ec5dd378117337bf95d14cc2c298b437cfda1780b436'],
+    }),
+    ('patsy', '0.5.2', {
+        'checksums': ['5053de7804676aba62783dbb0f23a2b3d74e35e5bfa238b88b7cbf148a38b69d'],
+    }),
+    ('dill', '0.3.4', {
```
Comment (boegel) [inline easybuild/easyconfigs/p/PyMC3/PyMC3-3.11.1-intel-2021b.eb:39]: We already have an easyconfig for `dill`, so also use proper dependency here?

**Review by boegel — CHANGES_REQUESTED** [review]

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
node3112.skitty.os - Linux UNKNOWN UNKNOWN, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/d1dc7ae3eae68d9790ecb3c4fe65b818 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
node3526.doduo.os - Linux UNKNOWN UNKNOWN, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/209f9d36b7f1f74704beb86663c7e3bf for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

**Comment by boegel** [comment]
Going in, thanks @ItIsI-Orient!

**Pull request merged**